doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,102 @@
+
+By using pre-qualified designs of products, you can omit the testing phase during your end product's qualification.
+
+All `Bluetooth product types`_ are used by Nordic Semiconductor, however, some of them can be qualified by yourself.
+All versions of End products, Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems, and Components are created and qualified by Nordic Semiconductor by default.
+Profile Subsystems (for example, :ref:`ug_bt_mesh` versions) can be qualified by any party.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the product.
+
+Qualification process
+*********************
+
+The product qualification cannot be inherited, hence, you must complete this process by yourself.
+You can either perform a full qualification process for a new product or edit an existing entry for a product that has already been qualified.
+
+Creating new entry
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+Updating existing entry
+=======================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You significantly changed an existing product's design so that the testing is required.
+* Your product can be added as a new model to one of your existing qualifications.
+* You switched to another SDK version for an existing product.
+
+.. note::
+
+   An existing entry can be edited only by the person that completed the listing.
+   If you want to change the entry owner, contact SIG support.
+
+Switching to another SDK version
+--------------------------------
+
+If you want to switch to another SDK version for an existing product, complete the following steps:
+
+1. Go to the `Nordic Semiconductor Infocenter`_ and search for the SoC or SiP model you use.
+
+#. Navigate to the relevant Compatibility Matrix directory.
+
+#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:
+
+  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
+     If yes, check the following:
```

Comment:
Could we switch the order? Does it matter from the process perspective which one is checked first (line 74 or line 81)? I think it would be better to keep step a and b as close to each other as possible given the sentence structure in the main step above.